core/client/app/controllers/setup.js

```diff
@@ -15,6 +15,11 @@ export default Ember.Controller.extend(ValidationEngine, {
     ghostPaths: Ember.inject.service('ghost-paths'),
     notifications: Ember.inject.service(),
 
+    stepOne: true,
```

Comment:
Totally valid.  I usually opt for the less specialized version for the sake of simplicity and future changes but if someone else preferred to use `match` it'd be fine by me.